js/index.js

```diff
@@ -21,63 +21,148 @@ let state = {
 
 // This function takes care of rendering the pizza based on the state
 // This function is triggered once at the beginning and every time the state is changed
-function renderEverything() {
+const renderEverything = () => {
   renderPepperoni();
   renderMushrooms();
   renderGreenPeppers();
   renderWhiteSauce();
   renderGlutenFreeCrust();
-
   renderButtons();
   renderPrice();
-}
-
-function renderPepperoni() {
-  document.querySelectorAll('.pep').forEach(onePep => {
-    if (state.pepperoni) {
-      onePep.style.visibility = 'visible';
-    } else {
-      onePep.style.visibility = 'hidden';
-    }
+};
+const toggleVisibility = (style, state) =>
+  state ? (style.visibility = "visible") : (style.visibility = "hidden");
+
+const renderPepperoni = () =>
+  document.querySelectorAll(".pep").forEach(onePep => {
+    toggleVisibility(onePep.style, state.pepperoni);
   });
-}
 
-function renderMushrooms() {
+const renderMushrooms = () =>
   // Iteration 1: set the visibility of `<section class="mushroom">`
-}
+  document
+    .querySelectorAll(".mushroom")
+    .forEach(mushroom => toggleVisibility(mushroom.style, state.mushrooms));
 
-function renderGreenPeppers() {
+const renderGreenPeppers = () =>
   // Iteration 1: set the visibility of `<section class="green-pepper">`
-}
+  document
+    .querySelectorAll(".green-pepper")
+    .forEach(greenPepper =>
+      toggleVisibility(greenPepper.style, state.greenPeppers)
+    );
 
-function renderWhiteSauce() {
+const renderWhiteSauce = () =>
   // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
-}
+  document
+    .querySelectorAll(".sauce")
+    .forEach(sauce => sauce.classList.toggle("sauce-white", state.whiteSauce));
 
-function renderGlutenFreeCrust() {
+const renderGlutenFreeCrust = () =>
   // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
-}
+  document
+    .querySelectorAll(".crust")
+    .forEach(glutenFreeCrust =>
+      glutenFreeCrust.classList.toggle(
+        "crust-gluten-free",
+        state.glutenFreeCrust
+      )
+    );
```

Comment:
On this both functions you could have used `querySelector` because there is just one element `.sauce` and one `.crust`.

`function renderWhiteSauce() { const oneSauce = document.querySelector(".sauce"); if (state.whiteSauce) { oneSauce.classList.add("sauce-white"); } else { oneSauce.classList.remove("sauce-white"); } }`

`function renderGlutenFreeCrust() { const oneCrust = document.querySelector(".crust"); if (state.glutenFreeCrust) { oneCrust.classList.add("crust-gluten-free"); } else { oneCrust.classList.remove("crust-gluten-free"); } }`